web-console/src/utils/druid-query.tsx

```diff
@@ -43,3 +43,61 @@ export async function queryDruidSql(sqlQuery: Record<string, any>): Promise<any[
   }
   return sqlResultResp.data;
 }
+
+function parseQueryRel(queryRel: string) {
```

Comment:
Can you use a more descriptive name than queryRel? maybe like `queryPlanResult` or whatever this is?